pandas/io/formats/latex.py

```diff
@@ -284,8 +286,13 @@ def _write_tabular_begin(self, buf, column_format: str):
             <https://en.wikibooks.org/wiki/LaTeX/Tables>`__ e.g 'rcl'
             for 3 columns
         """
-        if self.caption is not None or self.label is not None:
+        if (
+            self.caption is not None
+            or self.label is not None
+            or self.position is not None
+        ):
```

Comment:
As this test grows, it would be better to have ``self._table_float = any([p is not None for p in (caption, label, position)])`` inside ``__init__.py``, to be used both here and in ``_write_tabular_end``.